changedetectionio/flask_app.py

```diff
@@ -327,7 +316,7 @@ def login():
 
         password = request.form.get('password')
 
-        if (user.check_password(password)):
+        if (user.check_password(password, datastore=datastore)):
```

Comment:
I guess it could be done that way. I don't like putting globals in because it makes testing problematic. I don't even know how using `g` works with tests.

Reply:
I'll admit, i dont know much about this :) just an idea, yes, agreed